plugins/SlicerT/SlicerTUI.cpp

```diff
@@ -0,0 +1,193 @@
+/*
+ * SlicerTUI.cpp - controls the UI for slicerT
+ *
+ * Copyright (c) 2006-2008 Daniel Kauss Serna <daniel.kauss.serna@gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#include "SlicerTUI.h"
+
+#include <QDropEvent>
+#include <QFileInfo>
+
+#include "Clipboard.h"
+#include "DataFile.h"
+#include "Engine.h"
+#include "InstrumentTrack.h"
+#include "SlicerT.h"
+#include "Song.h"
+#include "StringPairDrag.h"
+#include "Track.h"
+#include "embed.h"
+
+namespace lmms {
+
+namespace gui {
+
+SlicerTUI::SlicerTUI(SlicerT* instrument, QWidget* parent)
+	: InstrumentViewFixedSize(instrument, parent)
+	, m_slicerTParent(instrument)
+	, m_noteThresholdKnob(this)
+	, m_fadeOutKnob(this)
+	, m_bpmBox(3, "21pink", this)
+	, m_snapSetting(this, "Slice snap")
```

Comment:
```suggestion
	, m_snapSetting(this, tr("Slice snap"))
```
You should use ``tr`` here if you are displaying this text. This goes for any other displayed strings in ``SlicerTUI``.

Reply:
Done